arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1115,91 +1146,88 @@
         },
         "capacity": {
           "format": "int64",
-          "description": "The number of units being provisioned. Range of values [For F1: 1-1, S1: 1-200, S2: 1-200, S3: 1-10]. To go above this range, call support.",
+          "description": "The number of provisioned IoT Hub units. See: https://docs.microsoft.com/azure/azure-subscription-service-limits#iot-hub-limits.",
           "type": "integer"
         }
-      }
+      },
+      "required": [
+        "name",
+        "capacity"
+      ]
     },
     "EventHubProperties": {
-      "description": "The properties of the provisioned event hub used by the Iot Hub.",
+      "description": "The properties of the provisioned Event Hub-compatible endpoint used by the IoT hub.",
       "type": "object",
       "properties": {
         "retentionTimeInDays": {
```

Comment:
If the user wants to specify the EventHubProperties, in the IotHubProperties as a part of IotHubDescription, then what are the minimal properties that he needs to specify? Right now everything is optional, which doesn't seem correct to me by intuition. If "retentionTimeInDays" and "partitionCount" are indeed optional then do they have any default values? 
- To put this question in a different way: What does the server do if the user specifies an empty EventHubProperties object `{ }`?
   - If the server sends a 400 bad request then it looks like something is required
   - If the server accepts it then it may be possible that it is assigning a default value to those properties. Can you please check the server behavior and confirm this?